src/main/scala/com/wavesplatform/state2/diffs/TransactionDiffer.scala

```diff
@@ -40,8 +42,8 @@ object TransactionDiffer {
         case mtx: MintingTransaction => MintingTransactionDiff(s, currentBlockHeight, settings, currentBlockTimestamp)(mtx)
         case cstx: ContendSlotsTransaction => ContendSlotsTransactionDiff(s,settings,currentBlockHeight)(cstx)
         case rstx: ReleaseSlotsTransaction => ReleaseSlotsTransactionDiff(s,settings,currentBlockHeight)(rstx)
-        case cctx: CreateContractTransaction => ContractTransactionDiff.create(s, currentBlockHeight)(cctx)
-        case ccstx: ChangeContractStatusTransaction => ContractTransactionDiff.changeStatus(s, currentBlockHeight)(ccstx)
+        case rctx: RegisterContractTransaction => RegisterContractTransactionDiff.apply(s, currentBlockHeight)(rctx)
```

Comment:
```.apply``` here should be moved due to the consistency of code style.

Reply:
yes, should be removed